### PR TITLE
Initiating release of 0.13.4

### PR DESCRIPTION
**Pull request by dfarrell07**
Initiating release of 0.13.4

### CODE REVIEW

**Review by nyechiel — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Review by skitt — CHANGES_REQUESTED** [review]
Can we get https://github.com/submariner-io/shipyard/pull/1146 in first?

**Comment by dfarrell07** [comment]
> Can we get https://github.com/submariner-io/shipyard/pull/1146 in first?

This is merged

File: releases/v0.13.4.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+version: v0.13.4
+name: 0.13.4
+branch: release-0.13
+status: shipyard
+components:
+  shipyard: 7b46174780e75440013788cd2858aca35b8d5ae0
```
Comment (skitt) [inline releases/v0.13.4.yaml]:
```suggestion
  shipyard: a3eee257fc83022342b49c0b959a17d29457f33e
```

**Review by skitt — CHANGES_REQUESTED** [review]

File: releases/v0.13.4.yaml (hunk 1 above)
Comment (dfarrell07) [reply releases/v0.13.4.yaml]: It's surprising that re-running the `make release` command doesn't result in updating this

File: releases/v0.13.4.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+version: v0.13.4
+name: 0.13.4
+branch: release-0.13
+status: shipyard
+components:
+  shipyard: a3eee257fc83022342b49c0b959a17d29457f33e
```
Comment (dfarrell07) [inline releases/v0.13.4.yaml]: Shouldn't this be https://github.com/submariner-io/shipyard/commit/32d409523c0f5ac0316650048e410284d0f1e2d5?

Comment (dfarrell07) [reply releases/v0.13.4.yaml]: I changed it to this, which I think is the tip of the release-0.13 branch.

Comment (skitt) [reply releases/v0.13.4.yaml]: Sorry, I was looking at the wrong branch!

**Review by skitt — APPROVED** [review]

**Pull request merged**